R/tiltIndicatorAfter-options.R

```diff
@@ -0,0 +1,87 @@
+#' tiltIndicatorAfter options
+#'
+#' @description
+#' These options are meant to be used mainly by developers or analysts while
+#' testing the code or creating data:
+#' * `tiltIndicatorAfter.co2_jitter_amount`: Controls the amount of random noise
+#' in the `co2*` columns.
+#' * `tiltIndicatorAfter.co2_keep_licensed_min_max`: Keeps the licensed `min`
+#' and `max` columns that yield the noisy `co2*` columns.
+#' * `tiltIndicatorAfter.co2_keep_licensed_min_max`: Controls verbosity.
+#'
+#' @keywords internal
+#' @name tiltIndicatorAfter_options
```

Comment:
@AnneSchoenauer @Tilmon 

FYI the options I show in the reprex are documented here. This generates an internal helpfile that is available but hidden from the website. Eventually we may elevate them to more user-friendly arguments in the `profile_emissions*()` functions.

Reply:
@maurolepore this looks great, thank you!! Just one thing I don't understand in the documentation you shared: 

> tiltIndicatorAfter.co2_keep_licensed_min_max`: Controls verbosity.

What do you mean with that and how does it differ to...

> `tiltIndicatorAfter.co2_keep_licensed_min_max`: Keeps the licensed `min`
#' and `max` columns that yield the noisy `co2*` columns.

Reply:
Great catch thanks! It was a typo. I meant to say: "`tiltIndicatorAfter.verbose`: Controls verbosity".

Reply:
Hi @maurolepore and @Tilmon thanks a lot for this - i think this option is great! Quickly what does verbosity mean?

Reply:
Verbose means "using more words than necessary to convey information" and it a common name for an option that makes things yell or shut up:

https://ropensci.org/blog/2024/02/06/verbosity-control-packages/